tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+from typing import Type, TypeVar
+
+import pytest
+
+from merkle_zeppelin import BinaryTree
+
+T = TypeVar("T")
+
+
+@pytest.fixture
```

Comment:
This fixture re-creates a class for each test, so it might be a good idea to move the class definition outside of the fixture or change the fixture scope to be created only once.

Reply:
totally agree, will do it for sure : )